release/models/bgp/openconfig-bgp-common-multiprotocol.yang

```diff
@@ -24,7 +24,16 @@ submodule openconfig-bgp-common-multiprotocol {
     for multiple protocols in BGP. The groupings are common across
     multiple contexts.";
 
-  oc-ext:openconfig-version "9.4.1";
+  oc-ext:openconfig-version "9.4.2";
+
+  revision "2023-10-13" {
+    description
+      "Fix revision '2023-03-31': send-community-type was added to the
+       wrong grouping.
+      Allow configuring send-community per AFI-SAFI at
+       neighbor/peer-group.";
+    reference "9.4.2";
+  }
```

Comment:
I think this should be version 9.5.0.   This is because we are adding a leaf-list here.    The minor revision would only be used to correct spelling or clarify a description (without changing the intended semantics).